conf/logback.xml

```diff
@@ -13,7 +13,7 @@
 
   <logger name="com.ning.http.client" level="${logger.com.ning.http.client:-INFO}"/>
 
-  <root level="DEBUG">
+  <root level="INFO">
```

Comment:
is this on purpose?

Reply:
yeah - I think I accidentally committed the DEBUG in my previous PR but that wasn't spotted - just resetting here.